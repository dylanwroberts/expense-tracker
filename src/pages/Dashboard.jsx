import React from 'react';
import Filters from '../components/Filters';
import Expenses from '../components/Expenses';

const Dashboard = () => (
    <div>
        <Filters />
        <Expenses />
    </div>
);

export default Dashboard;