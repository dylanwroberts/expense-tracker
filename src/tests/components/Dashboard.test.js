import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '../../pages/Dashboard';

test('should render the dashboard', () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
});