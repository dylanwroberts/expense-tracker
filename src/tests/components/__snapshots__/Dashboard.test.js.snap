// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dashboard 1`] = `
<div>
  <[object Object] />
  <Connect(Expenses) />
</div>
`;
